include/vcpkg/base/json.h

```diff
@@ -227,13 +227,13 @@ namespace vcpkg::Json
         Value& operator[](StringView key) noexcept
         {
             auto res = this->get(key);
-            vcpkg::Checks::check_exit(VCPKG_LINE_INFO, res, "missing key: \"%s\"", key);
+            vcpkg::Checks::check_exit(VCPKG_LINE_INFO, res != nullptr, "missing key: \"%s\"", key);
```

Comment:
Do you still want these edits with your shiny new SAL annotation?

Reply:
yeah, I don't like the implicit conversion of pointers to bool (although if you're against I'll remove the edits)

Reply:
I'm not against, just wanted to make sure it was intentional :)